lib/agenda/db-init.ts

```diff
@@ -11,9 +11,9 @@ const debug = createDebugger('agenda:db_init');
  * @param collection name or undefined for default 'agendaJobs'
  * @param cb called when the db is initialized
  */
-export const dbInit = function(this: Agenda, collection: string, cb?: (error: Error, collection: Collection<any> | null) => void) {
+export const dbInit = function(this: Agenda, collection: string | undefined, cb?: (error: Error, collection: Collection<any> | null) => void) {
   debug('init database collection using name [%s]', collection);
-  this._collection = this._mdb.collection(collection || 'agendaJobs');
+  this._collection = this._mdb.collection(collection ?? 'agendaJobs');
```

Comment:
this changes behaviour..
```
false || 'agendaJobs' results to "agendaJobs"
false ?? 'agendaJobs' results to false
```
is this intended?

Reply:
hmm, yeah I was thinking the same thing. I was aware that they don't behave the same.
I was thinking that back in the days people used "||" but to do more precisely "??"

So while changing I checked that all the unit-tests still run fine.

I was hoping to cleanup null vs undefined un-preciseness. 
By introducing this difference, I thought that going forward we are more aware when we want to use "undefined" and when we actually want to use "null".

But I think will go through the places and see, where mixing null/undefined will cause issues and where we use '??' to express default-options.